src/browser/api.ts

```diff
@@ -249,6 +249,27 @@ const webApi: IPCApi = {
       return Promise.resolve();
     },
   },
+  terminal: {
+    create: (params) => invokeIPC(IPC_CHANNELS.TERMINAL_CREATE, params),
+    close: (sessionId) => invokeIPC(IPC_CHANNELS.TERMINAL_CLOSE, sessionId),
+    resize: (params) => invokeIPC(IPC_CHANNELS.TERMINAL_RESIZE, params),
+    sendInput: (sessionId: string, data: string) => {
+      // Send via IPC - in browser mode this becomes an HTTP POST
+      void invokeIPC(IPC_CHANNELS.TERMINAL_INPUT, sessionId, data);
+    },
```

Comment:
This might be slow over the network. Fine for now, but we should probably allow the `wsManager` to send data for the IPC at some point.